concepticondata/concepticon.tsv

```diff
@@ -1723,7 +1723,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 1722	EIGHTY	Quantity	The cardinal number occurring after seventy-nine and before eighty-one, represented in Roman numerals as LXXX and in Arabic numerals as 80.	Number	
 1723	RESULT IN	Cognition	Having a specific result, a logical consequence.	Action/Process	
 1724	NINETY	Quantity	The cardinal number occurring after eighty-nine and before ninety-one, represented in Roman numerals as XC and in Arabic numerals as 90.	Number	
-1725	CORRECT (RIGHT)	Emotions and values	That are in accordance with fact.	Property	
+1725	CORRECT (RIGHT)	Emotions and values	To be in accordance with fact.	Property	
```

Comment:
Not only with fact, but also with what is expected.

Reply:
Ah, that might be a good way to distinguish it from TRUE.

Reply:
:smiley: